src/einsteinpy/plotting/geodesic/interactive.py

```diff
@@ -149,8 +149,21 @@ def plot(self, geodesic, color="#{:06x}".format(random.randint(0, 0xFFFFFF))):
             Hexcode (String) for the color of the
             dashed lines, that represent the Geodesic
             Picks a random color by default
-
+        title : str, optional
+            String for plot title text
+        aspect : str, optional
+            One of "auto", "data", "manual", or "cube"
+            Defaults to "auto"
+        aspect_ratio : dict, optional
+            Aspect ratio to define if aspect type is "manual"
+            Defaults to ``dict(x=1, y=1, z=1)``
+            
```

Comment:
Since there is a potential to raise a `ValueError`, it's better if we document it here [like so](https://numpydoc.readthedocs.io/en/latest/format.html#raises). Same goes for the other plot method.